deployments/cardano-prod.nix

```diff
@@ -0,0 +1,12 @@
+with (import ./../lib.nix);
+
+let
+  conf = { config, pkgs, resources, ... }: {
+    imports = [
+      ./../modules/datadog.nix
+    ];
+    services.dd-agent.tags = ["prod"];
+  };
+in {
+  report-server = conf;
+} // (genAttrs' (range 0 13) (key: "node${toString key}") (name: conf))
```

Comment:
Can we feasibly determine the node names from the deployment configuration rather than hardcoding them?

It's also unclear how `node0`, e.g., is sufficiently descriptive. We have multiple distinct `node0`s in different clusters reachable from the production deployer.